frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -552,34 +552,49 @@ export default ({
     },
     listenChatRoomActions ({ hash }) {
       // NOTE: this function should be synchronous function
-      // Or the listener couldn't catch the event because it could be emitted in advance
-      const isAddedNewMessage = (message: GIMessage): boolean => {
+      //       Or the listener couldn't catch the event because it could be emitted in advance
+      const isMessageAddedOrDeleted = (message: GIMessage) => {
         const { action, meta } = message.decryptedValue()
         const rootState = sbp('state/vuex/state')
-        const me = rootState.loggedIn.username
+        let addedOrDeleted = 'NONE'
 
         if (/.*(addMessage|join|rename|changeDescription|leave)$/.test(action)) {
           // we add new pending message in 'handleSendMessage' function so we skip when I added a new message
-          return { added: true, self: me === meta.username }
+          addedOrDeleted = 'ADDED'
+        } else if (/.*(deleteMessage)$/.test(action)) {
+          addedOrDeleted = 'DELETED'
         }
 
-        return { added: false, self: false }
+        return { addedOrDeleted, self: rootState.loggedIn.username === meta.username }
       }
 
       // we use this listener here so that we can get access to the GIMessage
       // it will be called immediately after this function is called as long as listenChatRoomActions is sync function
       sbp('okTurtles.events/once', hash, async (contractID, message) => {
         // NOTE: while syncing the chatroom contract, we should ignore all the events
         if (contractID === this.currentChatRoomId) {
+          const { addedOrDeleted, self } = isMessageAddedOrDeleted(message)
+
+          if (addedOrDeleted === 'DELETED') {
+            // NOTE: Message will be deleted in processMessage function
+            //       but need to make animation to delete it, probably here
+            const messageHash = message.decryptedValue().data.hash
+            const msgIndex = findMessageIdx(messageHash, this.messages)
+            document.querySelectorAll('.c-body-conversation > .c-message')[msgIndex]?.classList.add('c-disappeared')
```

Comment:
In dark-mode, the deleting animation uses a white background instead of a red one. Can you choose a nice red color so that the animation remains consistent in color?

Reply:
Oh, yes. Nice catch. I think the other colors (such as primary, danger, warning) are also strange in dark-mode.
![image](https://user-images.githubusercontent.com/57976479/230837076-b440f9d0-5672-4f75-ad82-83a58f54a8a1.png)

I think I should check the color palette.

Reply:
OK whenever @leihla gets to this we can update it later in a future PR

Reply:
We can create a new issue for this update on the color palette and do in another PR later.